### PR TITLE
Feature/mobx

### PR DESCRIPTION
**Pull request by Oleg0110**

### CODE REVIEW

File: front/src/stores/CreateListStore/CreateListStore.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { action, makeObservable } from "mobx"
+import { observer } from "mobx-react"
+
+class CreateListStore {
+   list = []
```
Comment (SaintAllary) [inline front/src/stores/CreateListStore/CreateListStore.js:5]: list -> lists

File: front/src/App.js [hunk 2]
```diff
@@ -1,18 +1,38 @@
...
-import Board from "./layouts/Board/Board"
-import Sidebar from "./layouts/Sidebar/Sidebar"
+import { Header } from "./layouts"
+import { Sidebar } from "./layouts"
+import { Home } from "./layouts"
+import { Projects } from "./layouts"
+import { Dashboards } from "./layouts"
+import { People } from "./layouts"
+import { Settings } from "./layouts"
+import { Create } from "./layouts"
```
[the remark this answers is not in the document]
Comment (Oleg0110) [reply front/src/App.js:10]: Corrected

File: front/src/stores/CreateListStore/CreateListStore.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import { action, makeObservable } from "mobx"
+import { observer } from "mobx-react"
+
+class CreateListStore {
```
[the remark this answers is not in the document]
Comment (Oleg0110) [reply front/src/stores/CreateListStore/CreateListStore.js:4]: Corrected

File: front/src/stores/CreateListStore/CreateListStore.js (hunk 1 above)
Comment (Oleg0110) [reply front/src/stores/CreateListStore/CreateListStore.js:5]: Corrected

File: front/src/App.js [hunk 4]
```diff
@@ -1,18 +1,38 @@
...
+         <BrowserRouter>
+            <Route exact path="/" component={Home} />
+            <Route exact path="/projects" component={Projects} />
+            <Route exact path="/dashboards" component={Dashboards}>
+               <Dashboards onClick={() => setIsListModalOpened(!isListModalOpened)}></Dashboards>
+               <CreateListModalWindow isModalOpened={isListModalOpened} onModalClose={() => setIsListModalOpened(false)} />
+            </Route>
+            <Route exact path="/people" component={People} />
+            <Route exact path="/settings" component={Settings} />
+            <Route exact path="/create" component={Create} />
```
[the remark this answers is not in the document]
Comment (Oleg0110) [reply front/src/App.js:34]: Corrected

**Pull request merged**